### PR TITLE
fixed small issue that prevented the refresh token from being used

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: tests/SessionTest.cs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ﻿using System;
 using AeroGear.OAuth2;
 using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
```
Comment (matzew) [inline tests/SessionTest.cs]: does this have benefits over nunit? Or would this MS framework be also compatible with Mono? just wondering :-)

Comment (edewit) [inline tests/SessionTest.cs]: Tried to compile it with mono, but there are just to many system dependent things in this project. Benefit of using this is that the test will run on the simulator.

Comment (matzew) [inline tests/SessionTest.cs]: nice!

On Wed, Jan 14, 2015 at 2:28 PM, Erik Jan de Wit <notifications@github.com>
wrote:

> In tests/SessionTest.cs
> <https://github.com/aerogear/aerogear-windows-oauth2/pull/3#discussion-diff-22935650>
> :
>
> > @@ -1,6 +1,9 @@
> >  ﻿using System;
> >  using AeroGear.OAuth2;
> >  using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
>
> Tried to compile it with mono, but there are just to many system dependent
> things in this project. Benefit of using this is that the test will run on
> the simulator.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-windows-oauth2/pull/3/files#r22935650>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by matzew** [comment]
@lfryc any comments ?

**Comment by lfryc** [comment]
@matzew I can't try this until I have working Visual Studio.

@sebastienblanc would you mind to try this?

**Comment by sebastienblanc** [comment]
on it

**Comment by sebastienblanc** [comment]
Okay it compiles but having a hard time to set up the demo when following https://github.com/aerogear/aerogear-windows-oauth2#example-usage , would need to ask @edewit for some assistance.

**Comment by matzew** [comment]
@edewit @lfryc @sebastienblanc what's the status here ?

**Comment by sebastienblanc** [comment]
:+1: Could run the demo app and upload my scary face to Google Drive

**Comment by sebastienblanc** [comment]
@edewit Could you rebase ?

**Comment by edewit** [comment]
@sebastienblanc  rebased

**Pull request merged**